example.py

```diff
@@ -32,13 +32,13 @@ def demo():
         return
 
     # Fetch datas
-    lvbx.system.get_deviceinfo()
+    lvbx.deviceinfo.get_deviceinfo()
```

Comment:
I think this was a bug

Reply:
yes, to know that I no longer have a livebox so it is complicated for me to maintain the code and test it

Reply:
Would you like to migrate the code to https://github.com/hacf-fr?